features/_usi/exec.post

```diff
@@ -6,7 +6,7 @@ rootfs="$1"
 
 mkdir -p "$rootfs/etc/gardenlinux"
 
-openssl x509 -in /builder/cert/gardenlinux-oci-sign.crt -pubkey -noout > "$rootfs/etc/gardenlinux/oci_signing_key.pem"
+openssl x509 -in /builder/cert/gardenlinux-nightly-oci-sign.crt -pubkey -noout > "$rootfs/etc/gardenlinux/oci_signing_key.pem"
```

Comment:
"make build" is called at least on workflows after the certificate files relevant are already renamed to not contain the prefix `gardenlinux-nightly-`. For future extensibility this feature should account a file just named `oci-sign.crt`. If the file might not exist:
- we should either check for the file and fail with missing prerequisites
- or add the code line above if and only if the file `oci-sign.crt` does not exist.